content/docs-beta/SUSHI/migration/_index.md

```diff
@@ -5,42 +5,91 @@ weight: 30
-As of the SUSHI 1.0 Beta release, the **ig.ini** file MUST use a template based on `fhir.base.template#current`. Specific template versions (i.e., other than `#current`) are expected to work in the future.  For now, any of the following should work:
+You must specify a template based on `fhir.base.template#current`. Older templates will not work with SUSHI 1.0. Any of the following should work:
+
   * `template = fhir.base.template#current`
   * `template = hl7.base.template#current`
   * `template = hl7.fhir.template#current`
   * `template = hl7.davinci.template#current`
   * `template = hl7.cda.template#current`
 {{% /alert %}}
 
-Update **sushi-config.yaml** to remove the `history` property if necessary. The `history` property is used by SUSHI to generate a **package-list.json** file, and its usage is now deprecated. Similar to `template` above, if SUSHI detects usage of the `history` property to generate a **package-list.json** file, it will log an error message containing the contents of the **package-list.json** that would be generated by the `history` property. Copy these contents and place them in a **package-list.json** file located at **customized-ig/package-list.json**.
+Another error message you might be concerns the `history` property in **sushi-config.yaml**. If you get that message, remove that property and create the **package-list.json** file as instructed.
+
+### 5 - Update .gitignore
+If you are using Git for version control, update your **.gitignore** file. 
 
-#### 6 - Update Version Control
-If you are using Git for version control, update your **.gitignore** file. In the example of **customized-ig**, the pre-migration **.gitignore** would have likely contained **ig.ini** and **package-list.json**, since those files were being generated by SUSHI, and the "true" versions were in the SUSHI configuration or **ig-data**. Now the versions that live at **customized-ig/ig.ini** and **customized-ig/package-list.json** are the "true" versions, and should therefore be removed from **.gitignore** and checked into version control. The **customized-ig/input** folder may have also been in the **.gitignore** file, since SUSHI previously generated output into that folder. Now that **.fsh** files are located in **customized-ig/input/fsh**, this folder must be removed from **.gitignore** and checked into version control.
+* Remove entries for **ig.ini**, **package-list.json**, and the **input** directory, if present. These two files and the entire **./input** folder should now be under source code control.
 
-SUSHI will now output to **customized-ig/fsh-generated**, so the **fsh-generated** folder should be added to the **.gitinore** file.
+* Add **./fsh-generated** to .gitignore. This is where SUSHI 1.0 puts all generated files. 
 
-#### 7 - Troubleshooting
+Here is a typical **.gitignore** suitable for SUSHI 1.0:
+
+```text
+.DS_Store
+Thumbs.db
+fsh-generated**
+input-cache**
+output**
+template**
+temp**
+```
```

Comment:
Actually, it should look more like this (which is the one that `sushi --init` creates):
```
.DS_Store
Thumbs.db
/fsh-generated
/input-cache
/output
/temp
/template
```

Reply:
There seems to be more than one way to skin the octocat.

Reply:
Fixed.